cypress/support/step_definitions/organisation/create/edit/edit.ts

```diff
@@ -0,0 +1,66 @@
+/// <reference types="cypress" />
+/// <reference types="../../../" />
+import {
+    After,
+    Before,
+    Given,
+    Then,
+    When
+  } from "cypress-cucumber-preprocessor/steps";
+  import { customAlphabet } from "nanoid";
+  import { isLoggedIn } from "../../../utils";
+  
+  const user = Cypress.env("USERNAME") || "admin";
+  let organisationId = "";
+  const nanoid = customAlphabet("abcdefghijklmnopqrstuvwxyz", 4);
+  
+  Given(/a (public) organisation exists/, type => {
+    cy.createOrganisation(organisationId, user, type === "public");
+  });
+  
+  Given("the user is on the edit organisation page", () =>
+    cy.visit(`/orgs/${organisationId}/edit/`)
+  );
+  
+  When(/the user selects "(.+)" public_access => {
```

Comment:
This is where the error you're getting occurs. This should be:

```suggestion
  When(/the user selects "(.+)"/, public_access => {
```